build.sbt

```diff
@@ -126,7 +126,10 @@ lazy val cats: Project = (project in file("integrations/cats"))
     libraryDependencies ++= Seq(
       "org.typelevel" %% "cats-core" % "2.1.1",
       scalaTest % Test,
-      scalaCheck % Test
+      scalaCheck % Test,
+      "org.typelevel" %% "discipline-scalatest" % "1.0.1" % Test,
+      "org.typelevel" %% "cats-laws" % "2.1.1" % Test,
+      "com.github.alexarchambault" %% "scalacheck-shapeless_1.14" % "1.2.5" % Test
```

Comment:
I would suggest using a shapeless-less dependency, like https://github.com/spotify/magnolify's scalacheck module

Reply:
In fact, for this one maybe just implementing the Arbitrary instance yourself would be justifiable :)